packages/start/routes.js

```diff
@@ -1,11 +1,11 @@
-import fg from "fast-glob";
-import fs from "fs";
-import path, { join } from "path";
-import { init, parse } from "es-module-lexer";
-import esbuild from "esbuild";
 import chokidar from "chokidar";
 import debug from "debug";
 import { dequal } from "dequal";
+import { init, parse } from "es-module-lexer";
+import esbuild from "esbuild";
+import fg from "fast-glob";
+import fs from "fs";
+import path, { join } from "path";
```

Comment:
Is this some VSCode extension or prettier plugin that changes the order of the imports? Maybe we should add it as `.vscode/extensions` so everyone follows it

Reply:
I think that's a built-in vscode settings I enabled

```json 
"editor.codeActionsOnSave": {
  "source.organizeImports": true
},
```

I think it used to not work with prettier before but they must have fixed it recently.

Reply:
Ohh awesome will enable it by default for the repo!